aldryn_reversion/admin.py

```diff
@@ -128,10 +128,16 @@ def post_clear_placeholder(self, request, placeholder):
 
         self._create_aldryn_revision(placeholder, request.user, comment)
 
-    def post_add_plugin(self, request, placeholder, plugin):
+    def post_add_plugin(self, request, *args):
+        # The signature for post_add_plugin
+        # prior to 3.3 was request, placeholder, plugin.
+        # On 3.3 this was changed to request, plugin
+        # to keep consistency with the other hook methods.
+        # So on both plugin is the last argument
+        plugin = args[-1]
```

Comment:
I get why we'd do this, but perhaps we should at least check to see if its a CMSPlugin?